_pages/team.md

```diff
@@ -46,13 +46,13 @@ He is multidisciplinary trained in Pre-clinical Medicine, Neural Engineering, Me
 ### Technology Evangelist-Pharma and Convenor
 #### @srija-ieee
 
-Dr Srija is the Co-Founder of AyeAI Consulting and Heads Company Operations and Pharma lab. She is a Director at Vyas Labs Pvt Ltd and Interglial Healthcare (OPC) Pvt Ltd. 
+Dr Srija is the Co-Founder of AyeAI Consulting and heads Company Operations and Pharma lab. She is a Director at Vyas Labs Pvt Ltd and Interglial Healthcare (OPC) Pvt Ltd. 
 
-She is a passionate Entrepreneur experienced in Startup strategy, New Product Road Map, Requirement Analysis, with skills in Artificial Intelligence for Pharma and Healthcare. She was one among the invitees of Roundtable “Transforming Primary Healthcare – Leveraging Technologies” at NASSCOM Technology and Leadership Forum 2019. She was an invited speaker for IEEE CIS Summer School 2018 and has delivered a handson session on topic “AI in Healthcare” Dr Srija is familiar with Course design, Seminars and Workshop delivery. 
+She is a passionate Entrepreneur experienced in Startup strategy, New Product Road Map, Requirement Analysis, with skills in Artificial Intelligence for Pharma and Healthcare. She was one among the invitees of Roundtable “Transforming Primary Healthcare – Leveraging Technologies” at NASSCOM Technology and Leadership Forum 2019. She was an invited speaker for IEEE CIS Summer School 2018 and has delivered a hands-on session on topic “AI in Healthcare.” Dr Srija is also familiar with Course design, Seminars and Workshop delivery. 
```

Comment:
The capitalizations still do not work out correct!

A single word capitalized make a word look like a noun. To emphasize a topic capitalize all words in that topic